lib/ActivityStreams.js

```diff
@@ -223,6 +223,7 @@ ActivityStreams.prototype = {
         baseUrl,
         `${baseUrl}#/`,
         `${baseUrl}#/timeline`,
+        `${baseUrl}#/timeline/bookmarks`
```

Comment:
This doesn't actually seem to be working but i'm assuming rlr is fixing this in https://github.com/mozilla/activity-streams/issues/223